app/helpers/reactjs_helper.rb

```diff
@@ -20,8 +20,8 @@ def select_requested_plugins(webpacked_plugin_ids, plugin_names)
   end
 
   def all_webpacked_plugins
-    Foreman::Plugin.registered_plugins.values.select do |plugin|
-      File.exist? File.join(plugin.path, 'webpack/index.js')
+    Foreman::Plugin.all.select do |plugin|
+      File.exist?(File.join(plugin.path, 'webpack/index.js')) || plugin.webpack_manifest_path
```

Comment:
Would it make sense to push the entire check into `plugin.rb`? i.e. have some function such as `uses_webpack?` that encapsulates whatever checks are needed to tell if a plugin uses webpack or not